docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_that_span_multiple_batches_using_evaluation_parameters.rst

```diff
@@ -0,0 +1,139 @@
+  - Configured a :ref:`Datasource <Datasources>` (or Datasources) with at least two Data Assets.
+  - Also created :ref:`Expectation Suites` for those Data Assets.
+  - Have a working :ref:`Evaluation Parameter Store`. (The default in-memory store from ``great_expectations init`` can work for this.)
+  - Have a working :ref:`Validation Operator`. (The default Validation Operator from ``great_expectations init`` can work for this.)
+
+Steps
+-----
+
+In a notebook, 
+
+#. **Import great_expectations and instantiate your Data Context**
+
+    .. code-block:: python
+
+        import great_expectations as ge
+        context = ge.DataContext()
+
+#. **Instantiate two Batches**
+
+    We'll call one of these Batches the *upstream* Batch and the other the *downstream* Batch. Evaluation Parameters will allow us to use Validation Results from the upstream Batch as parameters passed into Expectations on the downstream.
+    
+    It's common (but not required) for both Batches to come from the same :ref:`Datasource <Datasources>` and :ref:`BatchKwargsGenerator`.
+
+    .. code-block:: python
+
+        upstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_1"),
+            expectation_suite_name='my_expectation_suite_1'
+        )
+
+        downstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_2"),
+            expectation_suite_name='my_expectation_suite_2'
+        )
+
+#. **Define an Expectation using an Evaluation Parameter on the downstream Batch.**
+
+    .. code-block:: python
+
+        eval_param_urn = 'urn:great_expectations:validations:my_expectation_suite_1:expect_table_row_count_to_be_between.result.observed_value'
+        downstream_batch.expect_table_row_count_to_equal(
+            value={
+                '$PARAMETER': eval_param_urn, # this is the actual parameter we're going to use in the validation
+                '$PARAMETER.' + eval_param_urn: 10  # this is a *temporary* value so we can execute the notebook
+            }
+        )
+    
+    The core of this is a ``$PARAMETER : URN`` pair. When Great Expectations encounters a ``$PARAMETER`` flag during validation, it will replace the ``URN`` with a value retrieved from an :ref:`Evaluation Parameter Store` or :ref:`Metrics Store`.
+    
+    This declaration above includes two ``$PARAMETERS``. The first is the real parameter that will be used after the Expectation Suite is stored and deployed in a Validation Operator. The second parameter supports immediate evaluation in the notebook.
+
+    When executed in the notebook, this Expectation will generate an :ref:`Expectation Validation Result`:
+
+    .. code-block:: python
+
+        {
+            "result": {
+                "observed_value": 506
+            },
+            "meta": {},
+            "exception_info": null,
+            "success": false
+        }
+
```

Comment:
update to the new return format.

Reply:
This is what it returned when I ran it. I was using GE 0.11 or 0.11.1.

Reply:
I mean the format that will happen when you disable interactive evaluation. Since in that world, it will not be able to provide a result or success.

Reply:
Ah, I see. Okay, yes, I can make the change.

Observation: I'm not sure I saved the notebook (and data context, expectation suites, and data samples) I used to verify the behavior of this guide. So this'll take me some effort to reproduce. I'm kinda wishing that some far-seeing person had twisted my arm to add a reproducible test block when I first wrote the notebook.

Reply:
Made this change. Please review and resolve the conversation if you agree on the changes.